### ISSUE TITLE
GUI crashes after opening about 7 models

### ISSUE THREAD

**Issue by antoinefalisse** [opensim-org/opensim-gui#1439]
When opening and closing models to visualize IK results, the GUI crashes after about 7-8 times (opening model, visualizing motions, closing model, opening next model, etc). It happens with the rajagopal model for instance

**Comment by antoinefalisse** [comment]
Yes that is what is happening, but in the GUI. I only tried one model.

**Comment by AlbertoCasasOrtiz** [comment]
Donde ir os happening in both, maybe it is a memory leak in the way the visualizer loads the models or animations.

**Comment by aymanhab** [comment]
@antoinefalisse Is this Rajagopal model with muscles or without? if it reproduces with files in distribution/repo I can take a look.
